samza-core/src/main/java/org/apache/samza/operators/spec/WindowOperatorSpec.java

```diff
@@ -117,4 +121,16 @@ public WatermarkFunction getWatermarkFn() {
     FoldLeftFunction fn = window.getFoldLeftFunction();
     return fn instanceof WatermarkFunction ? (WatermarkFunction) fn : null;
   }
+
+  @Override
+  public Collection<StoreDescriptor> getStoreDescriptors() {
+    String storeName = getOpName();
+    String storeFactory = "org.apache.samza.storage.kv.RocksDbKeyValueStorageEngineFactory";
+
+    Serde storeKeySerde = new TimeSeriesKeySerde<>(window.getKeySerde());
+    Serde storeValSerde = window.getFoldLeftFunction() == null ? window.getMsgSerde() : window.getWindowValSerde();
+
+    StoreDescriptor descriptor = new StoreDescriptor(storeName, storeFactory, storeKeySerde, storeValSerde, storeName, Collections.emptyMap());
```

Comment:
Minor: Line width < 120 everywhere. You can turn on line end marker in IntelliJ to indicate overflow (Preferences -> Editor -> General -> Apperance -> Show right margin)

Reply:
useful! I've made line width consistent.